File: src/main/java/no/nav/service/pensjon/regler/to/VilkarsprovAlderpensjonForsteUttakRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;

import no.nav.domain.pensjon.regler.krav.Kravhode;

public class VilkarsprovAlderpensjonForsteUttakRequest extends ServiceRequest {
	

	private Kravhode kravhode;
	private Date fom;
	private int uttaksgrad;
	private double ft;
	private double ft67Soker;
	private double ft67SokerMinus1;
	private double ft67Virk;
	private double ft67VirkMinus1;


}

File: src/main/kotlin/no/nav/pensjon/regler/to/VilkarsprovAlderpensjonForsteUttakRequest.kt
package no.nav.pensjon.regler.to

import com.fasterxml.jackson.annotation.JsonAlias
import no.nav.pensjon.regler.domain.krav.Kravhode
import java.util.*
import java.time.LocalDate

class VilkarsprovAlderpensjonForsteUttakRequest : ServiceRequest() {
    var kravHode: Kravhode? = null
    @Deprecated("Use virkLd instead")
    @JsonAlias("fom")
    var virk: Date? = null
    @JsonAlias("fomLd")
    var virkLd: LocalDate? = null
    var uttaksgrad = 0
    var ft = 0.0
    var ft67Soker = 0.0
    var ft67SokerMinus1 = 0.0
    var ft67Virk = 0.0
    var ft67VirkMinus1 = 0.0
}